Installing-and-Configuring/configuring-keys.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: default
+title: Step 3 - Configuring-Keys
+permalink: /Configuring-Keys/
+nav_order: 7
+parent: Installing and configuring
+---
+
+<!-- markdownlint-disable MD033 -->
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+<!-- markdownlint-enable MD033 -->
+
+`/boot` missing
+===
+
+![Boot missing]({{ site.baseurl }}/images/boot_missing.jpg)
+
+If starting with a blank hard drive, the Heads setup will display an error
+ saying `/boot` cannot be found.  Unless there is an nonencrypted boot partition
+ already configured on the drive, select "No".  You can either use `fdisk` in
+ the recovery shell or using the partition management option during the OS
+ installation.
+
+Generating your PGP key
+===
+
+![GPG missing]({{ site.baseurl }}/images/gpg_missing.jpg)
+
+Select "Add a GPG key to the running BIOS" to enter the GPG Management menu.  If
+ you're using a new security key, you'll need to generate your key files. If you
+ already have the public key stubs for your security key, please proceed to
+ [Adding your PGP key](#adding-your-pgp-key).
+
+![GPG Management menu]({{ site.baseurl }}/images/gpg_management.jpg)
+
+Insert your security key and USB drive into the device, then chose "Generate GPG
+ keys manually on a USB security token".
+
+When at the GPG prompt, go into "Admin" mode and generate a new key inside the
+ security key:
+
+```shell
+admin
+generate
+```
+
+This will prompt you for the admin pin (`12345678` by default for Yubikey) and
+ then the existing pin (`123456` default for Yubikey).  Follow the other prompts
+ and eventually you should have a key saved onto the USB drive.
+
+When complete, you will be returned to menu.
+
+There is some more info in the [GPG guide](/GPG).
+
+Adding your PGP key
+===
+
+Heads uses your own GPG key to sign updates and as a result it needs the
+key stored in the ROM image before flashing the full Heads ROM.
+
+Ensure your security key and the USB drive with your key are still inserted.
+ Select  "Add a GPG key to the running BIOS" to enter the GPG Management menu,
+ then "Add a GPG key to the running BIOS + reflash".  Follow the steps and your
+ GPG key will be added to the Heads rom.
+
+Once `flashrom` is complete, reboot and now you should now be back in the Heads
+ runtime. It should display a message that is is unable to unseal TOTP.
+
+Because the reproducible flash has an empty MRC cache, you need to
+reboot one more time so that the PCR values as they would be going
+forward.
+
+Configuring the TPM
+===
+
+![TOTP/HOTP missing]({{ site.baseurl }}/images/otp_error.jpg)
+
+There aren't very many good details on how to setup TPMs, so this section could
+ use some work.
+
+Taking ownership
+---
+
+If you've acquired the machine from elsewhere, you'll need to establish physical
```

Comment:
This is unclear. From Heads FbWhiptail (now default) it is in Options, TPM, reset TPM.